cvcl/templates/index.html

```diff
@@ -39,10 +39,10 @@
         </p>
 
         <ol>
-            <li>Create course from Courses</li>
-            <li>Create environment definition from Environment Definitions</li>
-            <li>Create VM definition within Environment Definition</li>
-            <li>Create assignment from Assignments</li>
+            <li>Create course from <a href="{% url "courses" %}">Courses</a></li>
```

Comment:
This is inline help text. Remove it from here. It shouldn't be visible to students either. Add this back where appropriate according to #40.